docs/units/equivalencies.rst

```diff
@@ -217,7 +217,7 @@ here is an example::
     >>> omega_B = 2 * np.pi * beam_sigma**2
     >>> freq = 5 * u.GHz
     >>> u.Jy.to(u.K, equivalencies=u.brightness_temperature(omega_B, freq))
```

Comment:
While you are at it, could you add `# doctest: +FLOAT_CMP` to the end of this line, and just insert the real answer below? I think the dots are only confusing to people reading the documentation...